src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: "Avenir" ;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir" ;
```

Comment:
```suggestion
  font-family: "Avenir-heavy" ;
```
rename this font and use it where it have to be

Reply:
> 

Changes was done.Thanks for review!Corrects was done and uploaded.